src/gl/align-gl.cpp

```diff
@@ -138,7 +138,7 @@ void build_opengl_projection_for_intrinsics(matrix4& frustum,
     // additional row is inserted to map the z-coordinate to
     // OpenGL.
     matrix4 tproj;
-    tproj(0,0) = float(alpha);  tproj(0,1) = float(skew);  tproj(0,2) = 0.f;
+    tproj(0,0) = float(alpha);  tproj(0,1) = float(skew);  tproj(0,2) = float(u0);
```

Comment:
As mentioned in the comments of this function, the whole function is copied from https://jamesgregson.blogspot.com/2011/11/matching-calibrated-cameras-with-opengl.html . 
Looks like, there was a typo error while copying. I have manually compared and fixed this code. With this fix, alignment looks better.

Reply:
Nice catch!!